src/components/user-list.tsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-import { UserType } from '../pages/user-list-page';
+import { UserType } from '../queries/user-list-query';
 
 interface UserListProps {
-  users: UserType[];
+  users: UserType[] | undefined;
```

Comment:
Outra forma de escrever isso seria assim:
```suggestion
  users?: UserType[];
```